media-sound/zynaddsubfx/zynaddsubfx-3.0.6-r1.ebuild

```diff
@@ -42,11 +49,22 @@ BDEPEND="
 PATCHES=(
 	"${FILESDIR}"/${P}-docs.patch
 	"${FILESDIR}"/${P}-stdint.patch
+	"${FILESDIR}"/${P}-libzest_location.patch
+)
+MRUBY_ZEST_PATCHES=(
+	"${FILESDIR}"/${PN}-mruby-zest-build-${PV}-cflags_ldflags.patch
+	"${FILESDIR}"/${PN}-mruby-zest-build-${PV}-libzest_location.patch
+	"${FILESDIR}"/${PN}-mruby-zest-build-${PV}-makefile_find.patch
+	"${FILESDIR}"/${PN}-mruby-zest-build-${PV}-system_wide_location.patch
 )
 
 DOCS=( AUTHORS.txt NEWS.txt README.adoc )
 
 src_prepare() {
+	local i
+	for i in "${MRUBY_ZEST_PATCHES[@]}"; do
+		patch -d mruby-zest-build -Np1 -i "$i" || die
```

Comment:
Instead of calling patch directly here, I'd do a pushd/popd into the directory and use eapply.